pkg/Directory.Build.props

```diff
@@ -21,7 +21,7 @@
   <!-- nuspec properties -->
   <PropertyGroup>
     <Authors>Microsoft</Authors>
-    <PackageLicenseUrl>https://github.com/dotnet/machinelearning/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Similar comment here - our LICENSE file in the repo has a copyright in it. I think we should just be packaging that file - `$(RepoRoot)\LICENSE`